code/datums/gamemode/role/wizard_apprentice.dm

```diff
@@ -0,0 +1,69 @@
+/datum/role/wizard_master
```

Comment:
is this role even necessary? why not just use the wizard role inside the contract faction?

Reply:
Because non-wizards can also use the contract and it's not supposed to make them wizard. This role is the equivalent of the Greytide Leader for the implant.

Reply:
The somewhat annoying part of this is that it will give them the multirole icon for admins (as Greytide Leader does)